samples/bookinfo/src/reviews/reviews-wlpcfg/servers/LibertyProjectServer/.gitignore

```diff
@@ -0,0 +1 @@
+/apps/
```

Comment:
You can skip this change. I've already handled it in another PR. See below.

https://github.com/istio/istio/blob/5fcdb826dd8e4e08757c52a4587d0c499c1cafc5/samples/bookinfo/src/reviews/.gitignore#L1-L3